website/server/controllers/api-v3/auth.js

```diff
@@ -75,6 +75,10 @@ api.registerLocal = {
       email: {
         notEmpty: {errorMessage: res.t('missingEmail')},
         isEmail: {errorMessage: res.t('notAnEmail')},
+        notEmailDomains: {
```

Comment:
Why did you move the validation here instead of using a validate function on the model like I asked for?